benchmarks/cpp/tensorexpr/bench_gemm.cpp

```diff
@@ -81,16 +81,14 @@ BENCHMARK_DEFINE_F(Gemm, TensorExprTile32x32)(benchmark::State& state) {
   {
     auto const& loops = loop.getLoopStmtsFor(CT);
     te::For* m = loops[0];
-    te::For* mo;
     te::For* mi;
-    loop.splitWithMask(m, 32, &mo, &mi);
+    loop.splitWithMask(m, 32, &mi);
```

Comment:
Nit: we could remove `mi` here entirely since it's not used (the same in some other places).

Reply:
Done.